hbase/hbase.sh

```diff
@@ -76,36 +127,34 @@ EOF
       --configuration_file 'hbase-site.xml.tmp' \
       --name 'hbase.zookeeper.quorum' --value "${zookeeper_nodes}" \
       --clobber
-    bdconfig set_property \
-      --configuration_file 'hbase-site.xml.tmp' \
-      --name 'hbase.rootdir' --value "hdfs://${CLUSTER_NAME}-m:8020/hbase" \
```

Comment:
Removing this will cause HBase master being unable to start on standard and single node clusters. I will refactor this part of the code to be more readable.